swap_meet/vendor.py

```diff
@@ -1,2 +1,75 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self,inventory=None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self,new_item):
+        self.inventory.append(new_item)
+        return new_item
+
+    def remove(self,new_item):
+        try:
+            self.inventory.remove(new_item)
+            return new_item
+        except ValueError as error:
+            print(f"{new_item} not in inventory")
+            return False
+
+    def get_by_category(self,category):
+        category_list = []
+        try:
+            for item in range(len(self.inventory)):
+                if self.inventory[item].category is category:
+                    category_list.append(self.inventory[item])
```

Comment:
for the try part of your code, you don't necessarily have to use range and len you could do something like
```python
   for item in self.inventory:
            if item.category == category:
                category_list.append(item)